src/web/src/components/SearchInput/AuthorSearchInput.tsx

```diff
@@ -33,6 +33,8 @@ const AuthorSearchInput = () => {
   return (
     <>
       <input
+        // eslint-disable-next-line jsx-a11y/no-autofocus
```

Comment:
We removed the `autoFocus` property specifically because of the eslint fix that came in. The solution for this will have to come from a different source rather than adding the autoFocus prop back in. Remove this comment and the autoFocus prop from here.